frontend/server/controllers/RunController.php

```diff
@@ -430,7 +430,7 @@ public static function apiStatus(Request $r) {
 
         self::validateDetailsRequest($r);
 
-        if (!(Authorization::canViewSubmission($r->identity->identity_id, $r['submission']))) {
+        if (!(Authorization::canViewSubmission($r->identity, $r['submission']))) {
```

Comment:
```suggestion
        if (!Authorization::canViewSubmission($r->identity, $r['submission'])) {
```

igual en L578 y L619

Reply:
Listo